apps/qubit/modules/default/actions/fullTreeViewAction.class.php

```diff
@@ -271,7 +271,7 @@ protected function getNodeOrChildrenNodes($id, $baseReferenceCode, $children = f
         $titles[$key] = $node['text'];
       }
 
-      usort($data, function($el1, $el2) { return strnatcmp( $el1[‘text’], $el2[‘text’]); });
+      usort($data, function($el1, $el2) { return strnatcmp($el1['text'], $el2['text']); });
```

Comment:
Fixes a PHP warning that will become an error:

```
PHP Warning:  Use of undefined constant ‘text’ - assumed '‘text’' (this will throw an Error in a future version of PHP) in /atom/src/apps/qubit/modules/default/actions/fullTreeViewAction.class.php on line 274"
```

Reply:
Good catch!